accounting/overview/process_overview/vendor_bill.rst

```diff
@@ -0,0 +1,139 @@
+submenu under Purchases in the Accounting application. As a shortcut,
+you can also use the “New Bill” feature on the accounting dashboard.
+
+.. image:: ./media/vendor_bill01.png
+   :align: center
+
+To register a new vendor bill, start by selecting a vendor and inputting
+their invoice as the Vendor Reference, then add and confirm the product
+lines, making sure to have the right product quantities, taxes and
+prices.
+
+.. image:: ./media/vendor_bill02.png
+   :align: center
+
+Save the invoice to update the pre tax and tax amounts at the bottom of
+the screen.You will most likely need to configure the prices of your
+products without taxes as Odoo will compute the tax for you.
+
+.. note:: 
+    On the bottom left corner, Odoo shows a summary table of all taxes on the vendor bill. In several countries, different methods are accepted to round the totals (round per line, or round globally). The default rounding method in Odoo is to round the final prices per line (as you may have different taxes per product. E.g. Alcohol and cigarettes). However if your vendor has a different tax amount on their bill, you can change the amount in the bottom left table to adjust and match.
+
+Validate The Vendor Bill
+------------------------
+
+Once the vendor bill is validated, a journal entry will be generated
+based on the configuration on the invoice. This journal entry may differ
+depending on the the accounting package you choose to use.
+
+For most European countries, the journal entry will use the following
+accounts:
+
+-  **Accounts Payable: *defined on the vendor form***
+
+-  **Taxes: *defined on the products and per line***
+
+-  **Expenses: *defined on the line item product used***
+
+For Anglo-Saxon (US) accounting, the journal entry will use the
+following accounts:
+
+-  **Accounts Payable: *defined on the vendor form***
+
+-  **Taxes: *defined on the products and per line***
+
+-  **Goods Received: *defined* *on the product form***
+
+You can check your Profit & Loss or the Balance Sheet reports after
+having validated a couple of vendor bills to see the impact on your
+general ledger.
+
+Pay a bill
+----------
+
+To create a payment for an open vendor bill directly, you can Register a
+Payment at the top of the form.
+
+From there, you select the payment method (i.e. Checking account, credit
+card, check, etc…) and the amount you wish to pay. By default, Odoo will
+propose the entire remaining balance on the bill for payment. In the
+memo field, we recommend you set the vendor invoice number as a
+reference(Odoo will auto fill this field from the from the vendor bill
+if set it correctly).
+
+.. image:: ./media/vendor_bill03.png
+   :align: center
+
+.. note::
+    You can also register a payment to a vendor directly without applying it to a vendor bill. To do that, navigate to the payments submenu under the Purchases >. Then, from the vendor bill you will be able to reconcile this payment with directly.
```

Comment:
keep at 80 char per line and change the "navigate to the payments submenu under the Purchases" by a `:menuselection:`